manifests/params.pp

```diff
@@ -161,7 +161,7 @@
   $server_debuglevel                        = '3'
   $server_externalscripts                   = '/usr/lib/zabbix/externalscripts'
   $server_historycachesize                  = '8M'
-  $server_historyindexcachesize             = undef
+  $server_historyindexcachesize             = '4M'
```

Comment:
can the default stay at undef? Otherwise it would be a breaking change that requires a major release (instead of a minor).

Reply:
Sure can! I'll revert it.